### PR TITLE
fix magnet

### PR DESCRIPTION
**Pull request by mciparelli**
this attempts to fix https://github.com/butterproject/butter-desktop/issues/137

### CODE REVIEW

File: src/app/lib/streamer.js [hunk 1]
```diff
@@ -244,8 +244,7 @@
 
     var Streamer = {
         start: function (model) {
-            var torrent = model.get('torrent');
-            var torrentUrl = torrent.url ? torrent.url : torrent.magnet;
+            var torrentUrl = model.get('torrent');
```
Comment (xaiki) [inline src/app/lib/streamer.js]: did you test that this doesn't break archive.org support ?

**Comment by MrAdder** [comment]
Can you confirm this works @mciparelli ?

File: src/app/lib/streamer.js (hunk 1 above)
Comment (mciparelli) [inline src/app/lib/streamer.js]: I haven't, sorry. I just tested drag&dropping a torrent and a magnet link and both worked.

**Comment by mciparelli** [comment]
@MrAdder works with magnet and torrents for me.

**Pull request merged**